vnext/local-cli/runWindows/utils/build.js

```diff
@@ -73,55 +73,25 @@ async function nugetRestore(nugetPath, slnFile, verbose, msbuildVersion) {
 }
 
 async function restoreNuGetPackages(options, slnFile, verbose) {
-  let nugetPath =
-    options.nugetPath || path.join(os.tmpdir(), 'nuget.4.9.2.exe');
+  const nugetPath = path.join(os.tmpdir(), 'nuget.4.9.2.exe');
 
-  let downloadedNuget = false;
-  const ensureNugetSpinner = newSpinner('Locating NuGet executable');
   if (!(await existsAsync(nugetPath))) {
-    try {
-      nugetPath = execSync('where nuget')
-        .toString()
-        .trim();
-    } catch {}
-  }
-
-  if (!(await existsAsync(nugetPath))) {
-    await commandWithProgress(
-      ensureNugetSpinner,
-      'Downloading NuGet Binary',
-      'powershell',
-      `$progressPreference = [System.Management.Automation.ActionPreference]::SilentlyContinue; Invoke-WebRequest https://dist.nuget.org/win-x86-commandline/v4.9.2/nuget.exe -outfile ${nugetPath}`.split(
-        ' ',
-      ),
-      verbose,
+    const spinner = newSpinner('Downloading NuGet Binary');
```

Comment:
Why download it instead of using the version that's installed by VS 2019?

Reply:
I'm not sure of the intent of the original author. The existing code seems to only work against older NuGet versions though, so doing anything more comprehensive should likely be an independent change, since it will need some more validation. Similarly, I'm not sure if new versions from VS updates would break us here as well.